### PR TITLE
CPDP-1888: updates to accessibility statement

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -220,8 +220,8 @@ accessibilityStatement.contactingUs.p1=We provide a text relay service if you ar
 accessibilityStatement.contactingUs.p2=We can provide a British Sign Language (BSL) interpreter, or you can arrange a visit from an HMRC advisor to help you complete the service.
 accessibilityStatement.contactingUs.p3=Find out how to <a href="{0}">contact us</a>.
 accessibilityStatement.howWeTested.title=How we tested this service
-accessibilityStatement.howWeTested.p1=The service was tested on 6 December 2019 and checked for compliance with WCAG 2.1 AA. It was built using parts that were tested by the <a href="{0}">Digital Accessibility Centre</a>. Since then, it has been extended and the full service will be tested by HMRC within the next three months. Tests will include disabled users.
-accessibilityStatement.howWeTested.p2=This page was prepared on 6 April 2020. It was last updated on 6 April 2020.
+accessibilityStatement.howWeTested.p1=The service was tested on 15 May 2020 and checked for compliance with WCAG 2.1 AAA. It was built using parts that were tested by the <a href="{0}">Digital Accessibility Centre</a>.
+accessibilityStatement.howWeTested.p2=This page was prepared on 6 April 2020. It was last updated on 14 September 2020.
```
Comment (AndyHWChung) [inline conf/messages:224]: where did 14th September come from?

Comment (nubz) [reply conf/messages:224]: I put in Monday, presume it's got to be close to the day it's actually published - ticket says XXXX 2020

Comment (nubz) [reply conf/messages:224]: Steph has confirmed on the ticket to say 14 Sept is ok

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**